server.R

```diff
@@ -110,84 +134,43 @@ server <- function(input, output) {
                                measure_selection,
                                graph_type) {
     
-    # Measure names in drop down differ from those in data, use this to match them up
-    data <- if(measure_selection == "Total Population") {
-      filter(dataset, Measure == "Population.Index")
-      } else {
-        if(measure_selection == "Dependency Ratio") {
-          filter(dataset, Measure  == "Dependency.Ratio")
-          } else {
-            if(measure_selection == "Sex Ratio") {
-              filter(dataset, Measure  == "Sex.Ratio")
-            }
-          }
-      }
+    measure_title <- "Total Population Index"
```

Comment:
This needs to be set to equal the measure_selection input variable because when the function is used on tab 2 it will be dependent on the measure_selection input. I will update this